grammars/ini.cson

```diff
@@ -8,6 +9,14 @@
 'name': 'INI'
 'patterns': [
   {
+    'match': '\\s(\\d+)\\s'
+    'name': 'constant.integer.ini'
+  }
+  {
+    'match': '\\s(true|false)\\s'
```

Comment:
Same as the integer change above. We need to optionally support the whitespace here.